modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -18,8 +18,11 @@ import (
 	porttypes "github.com/cosmos/ibc-go/v8/modules/core/05-port/types"
 	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
+const invalidVersion = "invalid|version"
```

Comment:
I think there might be similar constants in the `ibctesting` pkg

Reply:
I cannot find one, but if you know where it is, I am glad to change this in a follow-up PR.

Reply:
I think some other tests use `InvalidID` to chuck into the version, but happy to keep it as is